plugins/tracker-resources/src/components/issues/IssuesList.svelte

```diff
@@ -305,7 +305,7 @@
     border-bottom: 1px solid var(--theme-button-border-hovered);
 
     &.mListGridChecked {
-      background-color: var(--theme-table-bg-hover);
+      background-color: var(--menu-bg-select);
```

Comment:
For table we have following styles.

&.checking { background-color: var(--theme-table-bg-hover); }
&.selected { background-color: var(--menu-bg-select); }

Let's use

Reply:
Done.